ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/anomalylocalization/CountSketch.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ *
+ */
+
+package org.opensearch.ml.engine.algorithms.anomalylocalization;
+
+import java.util.List;
+import java.util.Random;
+import java.util.stream.IntStream;
+
+import lombok.extern.log4j.Log4j2;
+
+/**
+ * Count sketch implementation.
+ */
+@Log4j2
+public class CountSketch implements Counter {
+
+    protected static final double INV_DELTOID = 1 / 0.01;
+    protected static final double LOG_BASE_2 = 2;
+    protected static final double INV_EPSILON = 1 / 0.001;
+
+    private int numHashes;
+    private int numBuckets;
+    private double[][] counts;
+    private int[] hashes;
+    private int[] signHashes;
+
+    /**
+     * Constructor.
+     */
+    public CountSketch() {
```

Comment:
`CountMinSketch` and `CountSketch` have some duplicate code. Is it possible to refactor to remove the duplicate, like adding one filed to indicate whether the `CountSketch` is `min` or not.